### PR TITLE
Token handler additional getters

### PR DESCRIPTION
**Pull request by AndyGura**

### CODE REVIEW

File: docs/modoc/TokenHandler.md [hunk 1]
```diff
@@ -152,30 +160,32 @@ func backlogFunds() : Nat
...
-### Function `totalUsableBalance`
+### Function `usableFunds`
 ``` motoko
-func totalUsableBalance() : Nat
+func usableFunds() : Nat
 ```
 
 retrieve the sum of all user usable balances. It's tricky to cache it
-because of excluding deposits, smaller than fee, from the usable balance
+because of excluding deposits, smaller than fee, from the usable balance.
```
Comment (timohanke) [inline docs/modoc/TokenHandler.md:188]: The last sentence can be removed from the function description so it doesn't appear in the docs. Can you move it into the implementation where the doc generator doesn't pick it up?

**Review by timohanke — COMMENTED** [review]
LG, just two small remarks

File: src/TokenHandler.mo [hunk 2]
```diff
@@ -554,28 +559,28 @@ module TokenHandler {
...
       totalDebited := values.4.0;
       totalCredited := values.4.1;
       journal.unshare(values.5);
     };
 
     func assertBalancesIntegrity() : () {
-      let usableSum = totalUsableBalance();
-      if (usableSum + fee_ * backlog.size() + totalDebited != consolidatedFunds_ + backlog.funds() + totalCredited) {
+      let usableSum = usableFunds();
+      if (usableSum + fee_ * backlog.size() + totalDebited != totalConsolidated_ + backlog.funds() + totalCredited) {
```
[the remark this answers is not in the document]
Comment (AndyGura) [reply src/TokenHandler.mo]: Done

File: docs/modoc/TokenHandler.md (hunk 1 above)
Comment (AndyGura) [reply docs/modoc/TokenHandler.md]: Done

**Comment by AndyGura** [comment]
Updated according to comments

File: src/TokenHandler.mo [hunk 3]
```diff
@@ -574,15 +574,13 @@ module TokenHandler {
 
     func assertBalancesIntegrity() : () {
       let usableSum = usableFunds();
-      if (usableSum + fee_ * backlog.size() + totalDebited != totalConsolidated_ + backlog.funds() + totalCredited) {
+      if (usableSum + fee_ * backlog.size() != backlog.funds() + creditedFunds()) {
```
Comment (timohanke) [inline src/TokenHandler.mo:577]: Also inline the usableFunds(), remove the let statement above

**Review by timohanke — APPROVED** [review]

**Pull request merged**